app/src/main/res/xml/account_settings_preferences.xml

```diff
@@ -42,5 +42,12 @@
             android:key="@string/pref_key_sync_logins"
             android:layout="@layout/checkbox_left_preference"
             android:title="@string/preferences_sync_logins" />
+
+        <androidx.preference.CheckBoxPreference
```

Comment:
Let's make this not visible by default so instead of it jittering and hiding it for non-nightly users on layout, we show it after layout for the nightly users `app:isPreferenceVisible="false"`